### PR TITLE
modified hosted_apps page, adding user config, updated navigation ins…

### PR DESCRIPTION
**Pull request by ellieseoejung**
…truciton

### CODE REVIEW

File: building/hosted-apps.md [hunk 1]
```diff
@@ -10,45 +10,42 @@ Hosted app pages are written in HTML and JavaScript, and do not require much cod
 
 For more granular control, it is possible to host your app with a third-party and use Transposit as a backend. These apps can interface with Transposit via our [JavaScript SDK](js-sdk.md) or by directly invoking [http endpoints](endpoints.md).
 
-## Modify the page template
```
Comment (esbie) [inline building/hosted-apps.md]: This entire "modify the page template" section (and the intro above it) should go under advanced

File: building/hosted-apps.md [hunk 2]
```diff
@@ -10,45 +10,42 @@ Hosted app pages are written in HTML and JavaScript, and do not require much cod
...
 
 ## Manage users
 
-You can view and administer registered users in **Authentication &gt; Registered Users**. You can delete users or their keys as necessary.
+You can view and administer registered users in **Users &gt; Registered Users**. You can delete users or their keys as necessary.
+
+## Advanced
+
+You can customize user settings schema for your users. Users can provide their key to work with your app.
+To check the changes have been applied correctly, go back to code editor, click Development under Auth & settings.
```
Comment (esbie) [inline building/hosted-apps.md]: this line and the lines below belong in Customize User Configuration

File: building/hosted-apps.md [hunk 3]
```diff
@@ -10,45 +10,42 @@ Hosted app pages are written in HTML and JavaScript, and do not require much cod
...
-3. Check the box for each data connection that should expect a user to provide their own key.
+1. Navigate to **Deploy &gt; Production Keys**.
+2. Click **Add key**
 
 ## Manage users
 
-You can view and administer registered users in **Authentication &gt; Registered Users**. You can delete users or their keys as necessary.
+You can view and administer registered users in **Users &gt; Registered Users**. You can delete users or their keys as necessary.
+
+## Advanced
```
Comment (esbie) [inline building/hosted-apps.md]: Advanced should go at the end of the page

**Comment by jpwain** [comment]
@ellieseoejung if you would, PR this to master branch instead of kmonkeyjam/11ty.

**Pull request closed without merging**